doc/_static/custom_scripts.js

```diff
@@ -0,0 +1,162 @@
+// Custom scripts for the Spyder docs site
+
+
+;(function () {
+
+    'use strict';
+
+    // Set the active image for the tour based on the element's class
+    function setActiveTourImage(activeElement) {
+        var activeClass = "tour-screenshot-active";
+        var classNames = activeElement.node.className.baseVal.split(" ");
+        for (var i = 0; i < classNames.length; i++) {
+            var imageToActivate = document.getElementById(classNames[i]);
+            if (imageToActivate) break;
+        };
+        var imageToDeactivate = document.getElementsByClassName(activeClass)[0];
+        imageToDeactivate.classList.remove(activeClass);
+        imageToActivate.classList.add(activeClass);
+    };
+
+    // Interactive tour driver
+    var driver = new Driver({
+        animate: false,
+        opacity: 0.1,
+        padding: 0,
+        allowClose: false,
+        onHighlightStarted: setActiveTourImage,
+    });
+
+    // Event handler to start tour
+    function startTour() {
+        driver.start();
+    };
+
+    // Interactive tour of Spyder for Quickstart using Driver
+    function setupTourDriver() {
+        driver.defineSteps([
+          {
+            element: "#introduction-rect",
+            popover: {
+              title: "Spyder",
+              description: 'Spyder is a powerful scientific IDE for Python. We will guide you through some of its most important features.',
```

Comment:
```suggestion
              description: 'Spyder is a powerful scientific IDE for Scientific Python. Here we will guide you through some of its most important features.',
```

* Emphasize that our focus is on Scientific Python.
* I'm not sure about `Here`, but I think it makes the second sentence a bit friendlier.